content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -70,15 +76,17 @@ spec:
   ...
 ```
 
-The pod spec takes precedence over the service account if both specify a `automountServiceAccountToken` value.
+If both the ServiceAccount and the pod spec specify a value for
+`automountServiceAccountToken`, the pod spec takes precedence.
```

Comment:
PodSpec is the object

Reply:
From a cluster operator's point of view, Pods have specifications. Pods are Kubernetes objects.
Maybe “the Pod's specification” or “the Pod's `.spec`”?

Reply:
OK